nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/AbstractPutHBase.java

```diff
@@ -64,6 +65,18 @@
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
 
+    protected static final PropertyDescriptor DEFAULT_VISIBILITY_STRING = new PropertyDescriptor.Builder()
+            .name("hbase-default-vis-string")
+            .displayName("Default Visibility String")
+            .description("When using visibility labels, any value set in this field will be applied to all cells that are written unless " +
+                    "an attribute with the convention \"visibility.COLUMN_FAMILY.COLUMN_QUALIFIER\" is present on the flowfile. If this field " +
+                    "is left blank, it will be assumed that no visibility is to be set unless visibility-related attributes are set. NOTE: " +
+                    "this configuration will have no effect on your data if you have not enabled visibility labels in the HBase cluster.")
```

Comment:
If user would like to set visibility label per column family, how would they do that? Default visibility string may be too broad but setting attributes for each column qualifier can be error-prone as those configuration need to be updated as new qualifier emerges. Isn't it beneficial to support column family level default visibility? Probably by user defined dynamic property can be used.